run.sh

```diff
@@ -146,16 +152,18 @@ else
 fi
 
 if [[ "$stages" == 'none' ]] ; then
-  echo "INFO: dont run any stages"
+  echo "INFO: don't run any stages"
   exit 0
 fi
 
-# In case if contrail folder is not bint to the container from host
-# it is needed to copy container builde on host to be able mount
-# data into the building conainers for build from sources.
-if [[ -n "$CONTRAIL_BUILD_FROM_SOURCE" && "$BIND_CONTRAIL_DIR" == 'false' ]] ; then
-  if [[ ! -e ${CONTRAIL_DIR}/contrail-container-builder ]] ; then
-    sudo docker cp $TF_DEVENV_CONTAINER_NAME:/root/contrail/contrail-container-builder ${CONTRAIL_DIR}/
+# In case if contrail folder is not bound to the container from host
+# it is needed to copy contrail sources and container builder on host to be able mount
+# data into the building containers for build from sources.
+CONTRAIL_SRC_VOLUME=$(docker volume ls | grep "${SRC_VOLUME_NAME}")
```

Comment:
SRC_VOLUME_NAME is empty for non src build case, so this grep always return something